decent-dao/fractal-interface
# Usul proposal events listeners

## Description

This PR implements chunk of event listeners for Usul-based proposal. 
It should cover pretty much all the interactions that are happening with proposal

## Notes
It also contains small fix for `proposalDescription` translation.

However, it does not contain page update when proposal state changed without even emitting. Like when voting timeline is passed (there's no specific event emitted, it is calculated via `state` func on `Usul` contract), as well when proposal goes from `Time Locked` state to `Executing` - there's no event/transaction/internal transaction happening. Possible workaround is adding `setTimeout` 🤔 

## Issue / Notion doc (if applicable)

Closes #804 
Closes #779 

## Testing

1. Create a proposal - you should see new proposal occured in the list shortly after creation. Proposal metadata should occur after that in a bit
2. Cast a vote on proposal. You should see your vote occured and breakdown change pretty much immediately after vote casted
3. Queue proposal. You should see status change reflection almost immediately
4. Execute proposal. You should see status change almost immediately after proposal execution

## Screenshots (if applicable)
